auto-scale-aws.sh

```diff
@@ -0,0 +1,90 @@
+#!/bin/bash
+
+# Restarts AWS instance and changes the volume
```

Comment:
Ambiguos use of the word "volume". Volume in aws means disk so we should change this to "instance type". Same below.

Reply:
Yep, forgot to change this. Fixed now.